### PR TITLE
Feature - preview

### PR DESCRIPTION
**Pull request by joefitter**
* Added preview component
* updated components to not be editable if in preview mode
* Added no conditions label

### CODE REVIEW

File: client/components/preview-licence.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+export default function NextSteps() {
+  const { isActionable, previewLink } = useSelector(state => state.application);
+  if (!isActionable) {
```
Comment (lennym) [inline client/components/preview-licence.js:6]: Is there some aspect of how this is set that limits it to ASRU users only?

Comment (joefitter) [reply client/components/preview-licence.js:6]: It's the same logic as the next steps button, so it shows when you are looking at the version in the task (not asru specific)

Comment (joefitter) [reply client/components/preview-licence.js:6]: do we want it to be asru only?

Comment (lennym) [reply client/components/preview-licence.js:6]: The ticket is titled "Give ASRU a preview of the draft licence for PPL applications" so it's somewhat ambiguous. My assumption was that it was meant to be exclusive though.

Comment (joefitter) [reply client/components/preview-licence.js:6]: ok cool I'll make it asru only for now - easy to reverse

File: client/components/conditions/conditions.js [hunk 2]
```diff
@@ -151,25 +151,27 @@ class Conditions extends Component {
...
-              content={content}
-              {...condition}
-              onSave={this.handleSave(condition.key)}
-              onRemove={this.handleRemove(condition.key)}
-            />
-          })
+          conditions.length === 0
+            ? <p><em>No {this.props.authorisations ? 'authorisations' : 'conditions'} added</em></p>
+            // ra conditions used to be added to the conditions array, we dont want to show them here.
+            : conditions.filter(c => !c.key.match(/^retrospective-assessment/)).map((condition, index) => {
```
Comment (lennym) [inline client/components/conditions/conditions.js:157]: Should this filter be applied before the length is checked ☝️?

Comment (joefitter) [reply client/components/conditions/conditions.js]: good point - updated

**Pull request merged**